src/caf/space/inputs.py

```diff
@@ -157,6 +161,59 @@ def _valid_data_col(cls, v, values):
         return v
 
 
+@dataclasses.dataclass
+class SpaceArguments:
+    """Command Line arguments for running space."""
+
+    config_path: Path
+    mode: str
+    out_path: Path
+
+    @classmethod
+    def parse(cls) -> SpaceArguments:
+        """Parse command line argument."""
+        parser = argparse.ArgumentParser(
+            description=__doc__, formatter_class=argparse.ArgumentDefaultsHelpFormatter
+        )
+        parser.add_argument(
+            "--mode",
```

Comment:
Adding first parameter `"-m"` (before `"--mode"`) will allow the user to specify using either `-m` or `--mode` to make arguments shorter